ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/Daemon/AssemblyReaderTest.fs

```diff
@@ -61,6 +61,9 @@ type AssemblyReaderCSharpTest() =
 
     override this.RelativeTestDataPath = "common/assemblyReaderShim"
 
+    [<FSharpLanguageLevel(FSharpLanguageLevel.FSharp80)>]
+    [<Test>] member x.``Attribute - Attribute usage 01 - Before F# 9``() = x.DoNamedTest()
+
     [<Test>] member x.``Attribute - Attribute usage 01``() = x.DoNamedTest()
```

Comment:
Test numbers duplicate.

Reply:
I consciously left it because it is the same test, but for different versions, so they have different name suffixes. But if it is necessary, I can change the number